deploy

```diff
@@ -4,6 +4,8 @@
 
 REPO=$TRAVIS_REPO_SLUG
 
+[[ -z $ITB_BRANCH_REGEXP ]] && ITB_BRANCH_REGEXP="itb\..*"
```

Comment:
`$ITB_BRANCH_REGEXP` only gets used in this file, right?

The test i see that uses it is `[[ "${TRAVIS_BRANCH}" =~ $ITB_BRANCH_REGEXP ]]`, which means it'll match anything that contains the string `itb.` - a regex match in bash matches any part of the string, so (1) the trailing `.*` in your regex is redundant, and (2) it will match even if `itb.` appears at the end of some other word.

You can fix this either by making the regex `^itb\.` or by using a glob pattern match instead; eg,

```
ITB_BRANCH_PATTERN="itb.*"

... [[ $TRAVIS_BRANCH = $ITB_BRANCH_PATTERN ]]
```